controllers/nodeagent.go

```diff
@@ -369,9 +371,35 @@ func (r *DPAReconciler) customizeNodeAgentDaemonset(dpa *oadpv1alpha1.DataProtec
 	if ds.Spec.RevisionHistoryLimit == nil {
 		ds.Spec.RevisionHistoryLimit = pointer.Int32(10)
 	}
+
+	if err := r.applyUnsupportedNodeAgentServerArgsOverride(dpa, nodeAgentContainer); err != nil {
+		return nil, err
+	}
+
 	return ds, nil
 }
 
+// Apply Override unsupported Node agent Server Args
+func (r *DPAReconciler) applyUnsupportedNodeAgentServerArgsOverride(dpa *oadpv1alpha1.DataProtectionApplication, nodeAgentContainer *corev1.Container) error {
```

Comment:
would refactor function created in https://github.com/openshift/oadp-operator/pull/1430 to be used here

It is extremely similar to this one. Adding annotation and sub command as parameters in it, it can be used in both places

Reply:
not GenerateCliArgsFromConfigMap, but [applyUnsupportedArgsOverride](https://github.com/openshift/oadp-operator/blob/master/controllers/velero.go#L150)

Reply:
and make the applyUnsupportedArgsOverride a public function living somewhere in common? Either way works for me.

Reply:
Done, updated the PR